docs/versioned_docs/version-0.x/intro.mdx

```diff
@@ -8,8 +8,29 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-# 
+#
```

Comment:
This space is required to prevent the default heading (which would be `intro` in this case) from showing up above the logo.

Reply:
good catch, ide was stripping trailing spaces